changelog.md

```diff
@@ -1,5 +1,13 @@
 # API changelog
 
+## 2.38.1 (DATE)
```

Comment:
I will update this with a new version number and the actual date at the time of release.